ci-operator/config/openshift/csi-driver-shared-resource/openshift-csi-driver-shared-resource-master.yaml

```diff
@@ -15,16 +15,19 @@ binary_build_commands: make build
 build_root:
   from_repository: true
 images:
-- dockerfile_path: Dockerfile
+- dockerfile_path: images/Dockerfile
```

Comment:
yeah @jkhelil you should copy what we do for jenkins

see https://github.com/openshift/release/blob/667722ec35fc0d6ef97846a12b81b49f76702223/ci-operator/config/openshift/jenkins/openshift-jenkins-master.yaml#L17-L18

you should have something like 
```
- context_dir: "images"
  dockerfile_path: Dockerfile
```

and the same for must gather and webhook images

Reply:
fixed